### ISSUE TITLE
Synchronize rotation series overlay tth width with the plane data's

### ISSUE THREAD

**Issue by psavery** [HEXRD/hexrdgui#1121]
Similar to the powder overlays. Modifying the rotation series tth width will modify the plane data's, and vice versa.

**Comment by joelvbernier** [comment]
I think this was closed by https://github.com/HEXRD/hexrdgui/pull/1125, wasn't it @psavery?

**Comment by psavery** [comment]
No. This will be easier to implement once we have overlays as classes instead of dicts. That is making good progress, though.

**Comment by cjh1** [comment]
This has been addressed
